packageclients/pkg/packageclient/interface.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Should this be 2022?

Reply:
I had originally written this file in 2021, I just moved it to a new directory. So I think date remains as it was.